quinn-proto/src/config/mod.rs

```diff
@@ -209,6 +209,10 @@ pub struct ServerConfig {
     /// rebinding. Enabled by default.
     pub(crate) migration: bool,
 
+    pub(crate) validation_token_lifetime: Duration,
+    pub(crate) validation_token_log: Arc<dyn TokenLog>,
+    pub(crate) validation_tokens_sent: u32,
```

Comment:
Should we wrap these fields up into a separate type? The repetition in the field names seems like a sign something isn't optimal...

Reply:
Sure--and this is nice because it gives me a place to write a high-level explanation of tokens and validation.